tests/phpunit/tests/locale.php

```diff
@@ -15,6 +15,38 @@ public function set_up() {
 		$this->locale = new WP_Locale();
 	}
 
+	/**
+	 * @dataProvider data_property_initializes_to_array
+	 * @ticket 57427
```

Comment:
Per [56070](https://core.trac.wordpress.org/ticket/56070), `@ticket` should precede `@dataProvider` and there should be a line break between them.

```suggestion
	 * @ticket 57427
	 *
	 * @dataProvider data_property_initializes_to_array
```

Reply:
Thanks @mukeshpanchal27, I've addressed this in https://github.com/WordPress/wordpress-develop/pull/3824/commits/bc268ba0fe8141f890580916469968288e66c437.

Reply:
Thanks @mukeshpanchal27. 

Please note, that proposal has not yet been adopted, as there's still discussion happening. I think it's okay to implement the changes you suggested here. Once the proposal is accepted and the handbook updated, then any  additional changes needed in this test method can be updated. Cheers